cellprofiler/modules/trackobjects.py

```diff
@@ -3085,4 +3204,13 @@ def upgrade_settings(self, setting_values, variable_revision_number,
             setting_values = setting_values + ["80", "40"]
             variable_revision_number = 6
 
+        # added after integration of FOLLOWNEIGHBORS
+        # TODO Lee: I am not sure if this (not from_matlab) needs to stay there
+        if (not from_matlab) and variable_revision_number == 6:
```

Comment:
It doesn't. You could also go ahead and remove any of the conditional blocks with `if from_matlab`.